keps/0006-stable-kafka-example.md

```diff
@@ -53,3 +65,65 @@ Here are some features and things we should be able to do to consider Kafka prod
 ### Non-Goals
 
 * Any new KUDO features will not be covered by this KEP
+
+### User Stories
```

Comment:
Are we also considering the k8s cluster operator who need to deploy and provide the ability to deploy kafka clusters to the kafka admin.
Also how about the dev/user who need access/to use to the kafka cluster?

Reply:
I'd file both of those under RBAC and out of scope for this KEP. We should document how to properly use Kubernetes RBAC for KUDO workloads though, after [KEP 0002](https://github.com/kudobuilder/kudo/blob/master/keps/0002-dynamic-crds.md)  is implemented. Can you create an issue?